index.html

```diff
@@ -174,26 +207,25 @@ <h2 class="attribute-name">Skills</h2>
                         <li>API Design</li>
 					</ul>
                 </div>
-            </li>
-        </ul>
     </div>
 </section>
 <section class="contact-section">
+    <div class="contact-wrapper">
     <div class="contact-header">
         <h2>
             I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h2>
     </div>
     <div class="contact-form-container">
         <form action="https://formspree.io/f/mrgvjkeo" method="post" class="contact-form">
-            
             <input name="name" type="text" class="full-name" placeholder="Full name" maxlength="30" required>
             <input name="email-address" type="email" class="email-address" placeholder="Email address" required>
             <textarea name="message" class="message" placeholder="Hey I would like to connect to..." maxlength="500" required></textarea>
             <input type="submit" value="Get in touch">
             
         </form>
     </div>
+</div>
     <div class="page-footer">
```

Comment:
- [ ] as shown in the figma design please implement the design structure and placement of the footer :+1: 
- figma template
![foot](https://user-images.githubusercontent.com/89055502/236343499-5a0a81c9-7c2b-4609-b736-abc610e8e094.png)